New package: BrazilCentralBank v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: BrazilCentralBank
- Repository: https://github.com/azeredo-e/BrazilCentralBank.jl
- Created by: @azeredo-e
- Version: v0.1.0
- Commit: 9187997893959faa7e1598726c4c9af8e35a280d
- Reviewed by: @azeredo-e
- Reference: https://github.com/azeredo-e/BrazilCentralBank.jl/issues/1#issuecomment-2000173979
- Description: BrazilCentralBank.jl is an interface in the Julia programming language for the data API of Brazil's Central Bank (BCB)
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598c97ec313a8a820724ca53979e13c6e14c4e0ba52987d15659d30688fd8a7583835f43540d7472e786a8f93897a10837b75810f2410e904901e1027ad4150826533abf77d070cbb59a65ad95c11d4b5f18433441231f4dfa69d00bb5285020da87f8e8a044d2d27eb4820dbbce044c73da7ccc4ccf73c5eb5ad71c0ce70e558a7 -->

goerz commented:
[noblock] Just a tip: `Documenter` should not be a dependency of your package, since it is only used for building the package documentation. Including it in the `[deps]` section of your main `Project.toml` means that anyone using this package also has to install `Documenter` and all of `Documenter`'s dependencies. You want to keep the main list of dependencies as small as possible. Instead, `Documenter` should only be a dependency in the `Project.toml` you use for building the documentation (most people use `docs/Project.toml`).

JuliaRegistrator committed:
New package: BrazilCentralBank v0.1.0

UUID: 1db8e8ca-f8ef-4fae-b3f5-ea2940ece087
Repo: https://github.com/azeredo-e/BrazilCentralBank.jl.git
Tree: 1f33427ffbbdfe8ecad4894158f84706790687a9

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
